docs/topics/mta-requirements-for-java-analyzer.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-requirements-for-java-analyzer_{context}"]
+= Adding minimum requirements for Java analyzer and discovery
+
+There is a minimum required for the Java analyzer, and as a result, the  discovery task.
+
+The minimum requirement is 1.5 GiB. However, this minimum requirement is not unadvisable. The suggested requirement is a minimum of 2 GiB.
```

Comment:
"is not unadvisable"?

I don't think this makes sense. I would just say the recommended and default option is 2GiB, but could be lowered to 1.5GiB though not recommended.